pkg/graph/schema.resolvers.go

```diff
@@ -413,6 +426,46 @@ func (r *planParticipantsAndProvidersResolver) ProviderLeaveMethod(ctx context.C
 	return providerLeaveTypes, nil
 }
 
+func (r *planPaymentsResolver) FundingSourceR(ctx context.Context, obj *models.PlanPayments) ([]model.FundingSource, error) {
+	panic(fmt.Errorf("not implemented"))
+}
+
+func (r *planPaymentsResolver) PayRecipients(ctx context.Context, obj *models.PlanPayments) ([]model.PayRecipient, error) {
+	panic(fmt.Errorf("not implemented"))
+}
+
+func (r *planPaymentsResolver) PayRecipientNote(ctx context.Context, obj *models.PlanPayments) (string, error) {
+	panic(fmt.Errorf("not implemented"))
+}
+
+func (r *planPaymentsResolver) PayType(ctx context.Context, obj *models.PlanPayments) (model.PayType, error) {
+	panic(fmt.Errorf("not implemented"))
```

Comment:
This is what I was noting, that GQL does not automatically map enums that are not in the models package. These should resolve themself if the enums are moved back to the models package.